src/mobile/apps/artwork/components/bid/index.jade

```diff
@@ -2,7 +2,7 @@
 if artwork.auction
   .js-artwork-auction-container
     .artwork-auction-bid-module
-      if artwork.auction.is_open && !artwork.auction.is_live_open && !artwork.is_sold && !helpers.moment().isAfter(artwork.auction.end_at)
+      if artwork.auction.is_preview || artwork.auction.is_open && !artwork.auction.is_live_open && !artwork.is_sold && !helpers.moment().isAfter(artwork.auction.end_at)
```

Comment:
Technically don't think we need the last check for time in this line, but don't want to change things unless absolutely necessary for the old page.